src/findNodeHandle.web.ts

```diff
@@ -0,0 +1,33 @@
+type GestureHandlerRef = {
+  viewTag: GestureHandlerRef;
+  current: HTMLElement;
+};
+
+export default function findNodeHandle(
+  viewRef: GestureHandlerRef | HTMLElement
+): HTMLElement | number | undefined {
```

Comment:
The `findNodeHandle` returns `null | number | React.Component<any, any> | React.ComponentClass<any>`.

Is there a reason why `undefined` is used here instead of `null`?